release: v1.53.0 (#5873)
apollographql/router: .changesets/fix_tninesling_demand_control_score_arguments.md

Reviewer: @tninesling please review this edited changeset

Author: @tninesling you may want to double check, but i accepted the suggestions as they looked good.

### Add argument cost to type cost in demand control scoring algorithm ([PR #5740](https://github.com/apollographql/router/pull/5740))

> When scoring operations in the demand control plugin, include field arguments in the type cost.

Reviewer: ```suggestion
The router's operation scoring algorithm for demand control now includes field arguments in the type cost.
```
